bindings/sampling-heap-profiler.cc

```diff
@@ -61,8 +61,15 @@ NAN_METHOD(StartSamplingHeapProfiler) {
     if (!info[1]->IsNumber()) {
       return Nan::ThrowTypeError("First argument type must be Integer.");
     }
+
+    #if NODE_MODULE_VERSION > NODE_10_0_MODULE_VERSION
```

Comment:
Preprocessor directives are typically not indented:
```
#if something
                         // some indented code
#endif
```

Reply:
Done.

Reply:
Did you push the changes? The preprocessor lines are still indented here.

Reply:
I think I misunderstood.
Now this should be fixed.